easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMakeCp'
+
+name = 'PRSice'
+version = '2.2.12'
+
+homepage = 'https://www.prsice.info/'
+description = """PRSice (pronounced 'precise') is a Polygenic Risk 
+Score software for calculating, applying, evaluating and 
+plotting the results of polygenic risk scores (PRS) analyses."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/choishingwan/PRSice/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1e3d43f269ad6c4c703431e3a96084e7c85391e00061684d60a8599d4e158c84']
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('binutils', '2.32'),
+    ('Eigen', '3.3.7', '', True),
+]
+
+files_to_copy = [(['bin/PRSice'], 'bin'), "README.md", "LICENSE"]
+
+sanity_check_paths = {
+    'files': ['bin/PRSice'],
+    'dirs': ['bin'],
```

Comment:
@sassy-crick Also checking for a non-empty `bin` directory when you're already checking for `bin/PRSice` is a bit silly, so let's trim this down to:

```python
sanity_check_paths = {
    'files': ['bin/PRSice'],
    'dirs': [],
}